packages/cli-plugins/ai-base/src/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Base AI plugin package for Fusion Framework CLI
+ *
+ * This package provides shared utilities and options for AI-related CLI plugins.
+ * It is an internal package and is not published to npm. Consuming plugins should
```

Comment:
`...internal package and is not published to npm.`. README says different: This package is published to npm but is **not intended for standalone use**.